qa/common/util.sh

```diff
@@ -512,17 +514,23 @@ remove_array_outliers() {
 
 function setup_virtualenv() {
     # Create and activate virtual environment
-    virtualenv --system-site-packages venv
-    source venv/bin/activate
-    pip install pytest
+    if [[ -v MSYSTEM ]]; then
+      pip3 install pytest
+    else
+      virtualenv --system-site-packages venv
+      source venv/bin/activate
+      pip install pytest
+    fi
 
     if [[ ${TEST_WINDOWS} == 1 ]]; then
-        pip3 install "numpy<2" tritonclient[all]
-    fi
+      pip3 install "numpy<2" tritonclient[all]
+    else
```

Comment:
```suggestion
    fi
```
The change seems intentional, but it gives me a syntax error and doesn't executing the install